exercises/twelve-days/canonical-data.json

```diff
@@ -0,0 +1,102 @@
+    },
+    {
+      "description": "sixth day six geese-a-laying",
+      "property": "verse",
+      "input": 6,
+      "expected": "On the sixth day of Christmas my true love gave to me, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "seventh day seven swans-a-swimming",
+      "property": "verse",
+      "input": 7,
+      "expected": "On the seventh day of Christmas my true love gave to me, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "eighth day eight maids-a-milking",
+      "property": "verse",
+      "input": 8,
+      "expected": "On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "ninth day nine ladies dancing",
+      "property": "verse",
+      "input": 9,
+      "expected": "On the ninth day of Christmas my true love gave to me, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "tenth day ten lords-a-leaping",
+      "property": "verse",
+      "input": 10,
+      "expected": "On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "eleventh day eleven pipers piping",
+      "property": "verse",
+      "input": 11,
+      "expected": "On the eleventh day of Christmas my true love gave to me, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "twelfth day twelve drummers drumming",
+      "property": "verse",
+      "input": 12,
+      "expected": "On the twelfth day of Christmas my true love gave to me, twelve Drummers Drumming, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "finds multiple verses, which are a fragment from the beggining of the song",
+      "property": "verses",
+      "input": [1, 3],
```

Comment:
hmm, sorry to swoop in with another  thing again, but I just remembered... what about assigning meaning to the two elements of this array? See how `beer song` does it at https://github.com/exercism/problem-specifications/blob/master/exercises/beer-song/canonical-data.json#L54-L55 . Any desire to do the same here?

Reply:
Added two elements instead of one array; named them `beginning` and `end` respectively. Not sure if it will be a good idea to split all cases into two groups, like "beer-song" does (for single and multiple verse test cases).

Reply:
should `beginning` and `end` be subkeys of input?
```
"input": {"beginning": 1, "end": 3}
```

Reply:
Should it? Wouldn't it be confused with some object or map data structure?

Reply:
> Should `beginning` and `end` be subkeys of input?

That would be an interesting change to make for the JSON Files all around. Then, instead of having to vary the input keys per exercise, test generators would know that all inputs can be found in `"input"`. That could be a useful change for the schema! I think the track maintainer who wishes that to happen for all JSON files should make the issue requesting it.